Formula/stringtie.rb

```diff
@@ -1,9 +1,9 @@
 class Stringtie < Formula
   # cite Pertea_2015: "https://doi.org/10.1038/nbt.3122"
   desc "Transcript assembly and quantification for RNA-Seq"
-  homepage "https://ccb.jhu.edu/software/stringtie"
-  url "https://ccb.jhu.edu/software/stringtie/dl/stringtie-1.3.6.tar.gz"
-  sha256 "9dbdf01ab3282dc6eb647409920b53f8e0df45f4c45faf9d6b5ca135a3738ee8"
+  homepage "https://github.com/gpertea/stringtie"
```

Comment:
Please revert this line. The official web site has not changed. See https://github.com/gpertea/stringtie#readme

Reply:
https://ccb.jhu.edu/software/stringtie redirects to insecure http, and I thought it was a cause of this error:
https://github.com/brewsci/homebrew-bio/runs/798027175#step:4:62
but now it seems not to be the case.

Reply:
https://ccb.jhu.edu/software/stringtie/ with the trailing slash works!